lib/GADS.pm

```diff
@@ -1329,6 +1329,85 @@ any ['get', 'post'] => '/settings/team_edit/:id' => require_any_role [qw/useradm
     };
 };
 
+any [ 'get', 'post' ] => '/settings/report_defaults/' => require_role 'superadmin'=> sub {
+    my $site             = var 'site';
+    my $logo             = $site->site_logo ? 1 : 0;
+    my $security_marking = $site->security_marking || config->{gads}->{header};
+
+    template 'layouts/page_reporting_overview' => {
+        page             => 'report_defaults',
+        page_title       => 'Report Defaults',
+        page_description => 'In this window you can set the default values for reports.',
+        back_url         => '/settings/',
+        logo             => $logo,
+        security_marking => $security_marking,
+        data_attributes => {
+            fileupload_url => '/settings/logo'
+        }
+    };
+};
+
+any [ 'get', 'post' ] => '/settings/logo' => require_login sub {
```

Comment:
The main reason it wasn't there was because when I discussed it with you previously, you said to put it in the main controller because you felt it wasn't an API call - I'll move it across.

Reply:
Ah, sorry - now that I've seen as such it makes sense to be as an API. Going forwards, in general anything returning a JSON response should be in /api.